CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 Change Log
 ==========
+Version 0.3.0 *(2019-02-04)*
+----------------------------
+
+* Prefill attachments  
+
+Bug Fixes:
+
+* Ensured all places the Beacon Config is accessed it has the overrides applied 
+* Crash when launching BeaconActivity with Context rather than Activity 
+* Crash when `lastThread` is null on `ConversationsAdapter`
+* Fix colors when button has low contrast
```

Comment:
Fix colors when button has low contrast
should be
Fix colors when a button has low contrast